source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -866,6 +905,23 @@ static ProcessingMode allDisabledMode() {
   return pm;
 }
 
+namespace {
```

Comment:
style nit: 
Instead of having multiple anonymous namespaces in this file, can we move those functions into one? And please move anonymous namespace to the top below  `namespace ExternalProcessing {`  or after using declaration if you need any.

Reply:
@tyxia would you prefer me to move funcs only that I have added or all the static ones that are in that file?

Reply:
I can also make some inits and merge functions static as they do not use *this.

Reply:
I have moved everything up into one anon namespace and along with moving out some functions out of the class members.